roles/swift-storage-1/tasks/juno.yml

```diff
@@ -71,7 +77,7 @@
   tags: swift
 
 - name: chwon srv/node
-  command: chown swift:swift -R /srv/node 
+  command: chown swift:swift -R /srv/node
```

Comment:
The [ACL](http://docs.ansible.com/ansible/acl_module.html) module could be used instead of a command here.